modules/monitoring/files/usr/lib/nagios/plugins/reload_service

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Script that NRPE runs when it's notified by the monitoring machine that an application
+# has crossed the high memory threshold.
+#
+set -eu
+
+APPNAME=$1
+
+logger --tag govuk_icinga_event_handler "Reloading service ${APPNAME} because monitoring has detected it using too much memory"
+
+sudo /sbin/initctl reload ${APPNAME}
```

Comment:
I think the `sudo` here is unneeded, as it's the `nagios` user we're allowing in the sudoers config. I'll raise a PR to remove.